packages/@aws-cdk/aws-appmesh/test/gateway-route.test.ts

```diff
@@ -1122,8 +1122,95 @@ describe('gateway route', () => {
           },
         },
       });
+    });
+  });
+
+  describe('with priority', () => {
+    test('setting priority using the method', () => {
```

Comment:
I don't think it makes sense to categorize tests on how the Route gets added.

I think a better idea is to have a separate test per Route type (so, one for `http`, one for `http2`, and one for `grpc`).

Reply:
I like this idea. I will work on sorting the tests as advised.